### PR TITLE
Image

### PR DESCRIPTION
**Pull request by esterlegaz**
Can you review everything is working properly?
Thanks in advance.

### CODE REVIEW

File: _src/assets/scss/components/_container_card-editor.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .page__main {
   @media all and (min-width: 768px) {
     display: grid;
-    grid-template-columns: 1fr 1frs;
```
Comment (japocoinyenhoang) [inline _src/assets/scss/components/_container_card-editor.scss:4]: change 1 frs, and white instead 1 fr.

File: _src/assets/js/main.js [hunk 2]
```diff
@@ -86,4 +86,31 @@ function applyPalette(e) {
...
+const divPreviewImage = document.querySelector('.square__img');
+
+function getImage(e){
+  const myFile = e.currentTarget.files[0];
+  fr.addEventListener('load', writeImage);
+  fr.readAsDataURL(myFile);
+}
+
+function writeImage() {
+  console.log('Hola');
```
Comment (japocoinyenhoang) [inline _src/assets/js/main.js]: Do we really need a hi!! message from the console?

File: _src/assets/js/main.js [hunk 3]
```diff
@@ -86,4 +86,31 @@ function applyPalette(e) {
 // En este caso, no se porque... no me funciona con 'click' y si con 'change'
 for (const c of colors) {
   c.addEventListener('change', applyPalette);
-}
\ No newline at end of file
+}
+
+// Añadir imagen
+
```
Comment (japocoinyenhoang) [inline _src/assets/js/main.js:91]: Perfect job girls, very smart.
The olny thing is when I try to choose a image to upload, it ask me for a double confirmation and I don't know if it should do that or not.

**Review by japocoinyenhoang — COMMENTED** [review]
Hi girls, 
First of all, congratulation, is a very smart job.
I do review the pull request and it works.
Thanks for the hi!! message from the console.log, but i think it is not neccesary.
And also I don't know if it happends when you try to upload the pic, but when i tried, it ask me to do a dobble confirmation, and I don't know it should me like that or not.

Once more time, thanks, and good job.

File: _src/assets/js/main.js (hunk 3 above)
Comment (laurag94) [reply _src/assets/js/main.js:91]: I reviewed it and it works fantastic! You can merge it

Comment (esterlegaz) [reply _src/assets/js/main.js:91]: Thanks both of you :)
Let's see if it works for @babelarr so we can definitely merge it.

File: _src/assets/js/main.js (hunk 2 above)
Comment (esterlegaz) [reply _src/assets/js/main.js]: We forgot about it haha
I've already deleted it.
Thanks.

File: _src/assets/js/main.js (hunk 3 above)
Comment (babelarr) [reply _src/assets/js/main.js:91]: It's perfect!

**Review by babelarr — APPROVED** [review]

**Review by laurag94 — APPROVED** [review]

**Review by japocoinyenhoang — APPROVED** [review]

**Pull request merged**